devito/ir/iet/visitors.py

```diff
@@ -1229,7 +1229,10 @@ def visit_Iteration(self, o):
         dimension = uxreplace(o.dim, self.mapper)
         limits = [uxreplace(i, self.mapper) for i in o.limits]
         pragmas = self._visit(o.pragmas)
+
         uindices = [uxreplace(i, self.mapper) for i in o.uindices]
+        uindices = filter_ordered(i for i in uindices if isinstance(i, Dimension))
```

Comment:
I presume you're liable to get `int` or some other type that doesn't have the attribute `is_Dimension` here?

Reply:
yes, whatever the caller passes in `self.mapper`